static/elements/chromedash-feature.js

```diff
@@ -29,23 +29,18 @@ class ChromedashFeature extends LitElement {
     };
   }
 
-  constructor() {
-    super();
-    this.open = false;
-  }
-
   firstUpdated() {
     this._initializeValues();
   }
 
-  updated(props) {
+  shouldUpdate(props) {
```

Comment:
You shouldn't do any side-effects in `shouldUpdate`. Why did this change from `updated`? If it's to go before the first render, then override `update()` and do the work before the `super.update(changedProperties)` call.

Reply:
>  it's to go before the first render, then override `update()` and do the work before the `super.update(changedProperties)` call.

Exactly. Thanks for the suggestion. Applied.